src/Calculator/Views/HistoryList.xaml

```diff
@@ -66,13 +66,15 @@
                                    Margin="0,0,0,4"
                                    Style="{ThemeResource BodyTextBlockMediumStyle}"
                                    AutomationProperties.Name="{x:Bind AccExpression}"
+                                   IsTextSelectionEnabled="True"
                                    Text="{x:Bind Expression}"
                                    TextAlignment="Right"
                                    TextWrapping="Wrap"/>
                         <TextBlock x:Name="resultTextBlock"
                                    Style="{ThemeResource TitleTextBlockStyle}"
                                    FontWeight="SemiBold"
                                    AutomationProperties.Name="{x:Bind AccResult}"
+                                   IsTextSelectionEnabled="True"
```

Comment:
The `exprTextBlock` and `resultTextBlock` both stretch across the width of the ListViewItem.  After text selection is enabled, this makes it extremely difficult to click on the item itself, as the only available hit target is the thin margin between the two textblocks.
![image](https://user-images.githubusercontent.com/23107231/54320094-0b20cf80-45a9-11e9-9476-b00d3b8d4b3e.png) 

We should apply `HorizontalAlignment="Right"` to both textblocks here, so as to minimize their hit target while still enabling text selection when they are explicitly hovered.